bin/install

```diff
@@ -18,23 +19,109 @@ error_exit() {
   exit "${2:-1}"
 }
 
-platform() {
+get_platform() {
   local -r os=$(
     case "$(uname | tr '[:upper:]' '[:lower:]')" in
       darwin) echo "osx" ;;
       *) echo "linux" ;;
     esac
   )
 
-  echo "${os}-amd64"
+  echo "${os}"
```

Comment:
Since this function is no longer responsible of directly choosing the release asset to be installed, it now only returns the platform name, either `osx` or `linux`